src/views/Lottery/components/TicketCard/BuyTicketModal.tsx

```diff
@@ -93,6 +93,7 @@ const BuyTicketModal: React.FC<BuyTicketModalProps> = ({ max, onDismiss }) => {
           {TranslateString(462, 'Cancel')}
         </Button>
         <Button
+          id="Lottery-Buy-Complete"
```

Comment:
Do you prefer capital letters? If not, can we make these lowercase to maintain consistency with the other other stuff.